### DISCUSSION TITLE
How to verify if ssl certificate is created?

### DISCUSSION THREAD

**Discussion by Lancelee92** [shibayan/appservice-acmebot#358] [Q&A] [answered]
Currently i am using REST api to create certificate for my free tier app service, 
the AddCertificate POST request is working with status 202 and location with url. When i try with GET request with the returned url, it did return status code 200.
But when i browse the custom domain (domain.xyz) for my app service, its still running on ssl certificates for *.azurewebsites.net.

Is there anything i did wrong for the post request ?

**experiment** is my resource group name,
**dummy-cert** is my app name,
i do not have any deployment slots as i am using free tier, but i still put development since it is required.
`
{
  "ResourceGroupName": "experiment",
  "AppName": "dummy-cert",
  "SlotName": "development",
  "DnsNames": [
    "domain.xyz",
    "www.domain.xyz"
  ],
  "UseIpBasedSsl": false,
  "ForceDns01Challenge": false
}
`

How can i verify that the ssl certificate has been created and how can i check if it is working correctly?
is there any additional settings that i need to set?

**Comment by shibayan** [comment, marked as answer]
Make sure that the custom domain is mapped to the Web App, as it is not supported by the free tier.
https://docs.microsoft.com/en-us/azure/app-service/app-service-web-tutorial-custom-domain

**Reply by Lancelee92** [reply]
Thanks for replying, i had already upgraded the plan to shared plan, and after some setup, i manage to browse the site with my custom domain (domain.xyz), just that the cert for domain.xyz is still using azure cert.

I dont really understand how to link the url in the location header with this domain.xyz.

Really hope you can show me if possible. Thanks in advance.

**Reply by shibayan** [reply]
You cannot use custom SSL with the shared plan; you need a Basic or higher plan.
https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#prepare-your-web-app
